backend/src/repos/getColumnNames.js

```diff
@@ -0,0 +1,11 @@
+import { db } from '../data/connection';
+
+export const getColumnNames = async table => {
+  const columns = await (await db.query(`show columns from ${table};`)).results;
+
+  await columns.forEach((column, index) => {
+    columns[index] = column.Field;
+  });
+
+  return columns;
+};
```

Comment:
@kkeszthe oooh, I see what you did there :D
no, this can be in the code, you'll have to modify the repo significantly if there are changes in the db because of this, with my suggestion it'll work, just maybe it won't use the new columns